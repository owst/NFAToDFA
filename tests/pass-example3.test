0
0--a->1,2
1--a->1
1--b->2
1
