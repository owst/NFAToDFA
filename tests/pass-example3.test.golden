digraph NFA {
rankdir = LR;
node [shape = point]; init
0 [label="0" shape="circle" ]; init -> 0;
1 [label="1" shape="doublecircle" ];
0 -> 1 [label="a"];
1 -> 1 [label="a"];
2 [label="2" shape="circle" ];
1 -> 2 [label="b"];
0 -> 2 [label="a"];
}
